src/middlewares/loggers/error.logger.ts

```diff
@@ -0,0 +1,27 @@
+import { Request, Response } from 'express';
+import morgan, { StreamOptions } from 'morgan';
+
+import logger from '../../config/winston.config';
+
+const stream: StreamOptions = {
+    write: (message) => logger.error(message)
+};
+
+morgan.token('response-body', ({}, res: Response) => {
```

Comment:
Consider keeping configuration in a separate "morgan" service and defining tokens in constants before using them across the app to avoid repetition.

Reply:
Great idea! I've now added this as a 'HttpLogger' service. I tried to follow DI, but I had to pass the tokens on the express loader itself as I couldn't seem to find a way to get it to load the tokens from the service itself.